### PR TITLE
Add 1x1 specialization for conv with NCHW order

### PR DESCRIPTION
**Pull request by xiaomengy**
Summary: Add 1x1 specialization for conv with NCHW order

Differential Revision: D8944686

### CODE REVIEW

**Comment by BIT-silence** [comment]
Some benchmark on CPU with N = 128, C = 128, H = 32, W = 32, M = 128, G = 4,
original impl: about 250ms
current impl: about 10ms

File: caffe2/operators/batch_matmul_op.h [hunk 1]
```diff
@@ -255,7 +255,7 @@ class BatchMatMulOp final : public Operator<Context> {
 
       // TODO(T23893772): doing this in a loop is likely going to be slow on GPU
```
Comment (yinghai) [inline caffe2/operators/batch_matmul_op.h]: Does this comment still apply?

**Review by yinghai — COMMENTED** [review]
Could you provide some comment on what the difference is between `batchedgemm` and `batchedstridedgemm`?

File: caffe2/operators/batch_matmul_op.h (hunk 1 above)
Comment (BIT-silence) [reply caffe2/operators/batch_matmul_op.h]: Actually I'm not sure about that. For the batmatmul op, we will update it later to also support broadcasting

**Review by houseroad — APPROVED** [review]
LGTM.

If using OSS, we can always do git clang-format on the staging files.
Mercurial may also support this.

**Pull request closed without merging**